mypy/main.py

```diff
@@ -112,8 +112,11 @@ def process_options() -> Tuple[List[BuildSource], Options]:
             parsed flags)
     """
 
+    # Make the help output a little less jarring.
+    help_factory = (lambda *a, **k:
+                    argparse.RawDescriptionHelpFormatter(max_help_position=28, *a, **k))
```

Comment:
> I assume it's because the ArgumentParser constructor also passes in a `max_help_position`.
I don't see that in the argparse.py source code. My theory is that it's because mypy is naive when combining explicit keyword args with **k.

> "Only the name of this class is considered a public API."
So technically they might as well have used an enum...

Anyway, I will make the mypy error go away by defining a lambda that just takes a `prog` argument.